<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="KARMA" ignored="0" name="Rerunning failed tests" tests="6" failures="2" timestamp="2020-12-28T19:03:18 CST" time="0.429" errors="0">
  <testcase name="test1" time="0.1" classname="org.testng.rerunningfailedtests.RerunFailedTestsFromResults"/>
  <testcase name="test2" time="0.007" classname="org.testng.rerunningfailedtests.RerunFailedTestsFromResults">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: null
at java.util.ArrayList.forEach(Unknown Source)
... Removed 30 stack frames]]>
    </failure>
  </testcase> <!-- test2 -->
  <testcase name="test3" time="0.003" classname="org.testng.rerunningfailedtests.RerunFailedTestsFromResults"/>
  <testcase name="test4" time="0.005" classname="org.testng.rerunningfailedtests.RerunFailedTestClass2">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: null
at java.util.ArrayList.forEach(Unknown Source)
... Removed 30 stack frames]]>
    </failure>
  </testcase> <!-- test4 -->
  <testcase name="test5" time="0.007" classname="org.testng.rerunningfailedtests.RerunFailedTestClass2"/>
  <testcase name="test6" time="0.005" classname="org.testng.rerunningfailedtests.RerunFailedTestClass2"/>
</testsuite> <!-- Rerunning failed tests -->
